src/Home/index.tsx

```diff
@@ -82,198 +86,263 @@ const Home: FunctionComponent = () => {
               <SvgXml
-                xml={topIcon}
-                width={TOP_ICON_SIZE}
-                height={TOP_ICON_SIZE}
-                fill={topIconFill}
-                accessible
-                style={style.topIcon}
-                accessibilityLabel={topIconAccessibilityLabel}
+                xml={statusIcon}
+                width={STATUS_ICON_SIZE}
+                height={STATUS_ICON_SIZE}
+                fill={statusIconFill}
+                style={style.statusIcon}
               />
-              {appIsActive && <ExpandingCircleAnimation />}
+              {exposureDetectionStatus && <ExpandingCircleAnimation />}
             </View>
-            <Text style={style.headerText} testID={"home-header"}>
-              {headerText}
-            </Text>
-            <Text style={style.subheaderText} testID={"home-subheader"}>
-              {subheaderText()}
-            </Text>
           </View>
-        </GradientBackground>
-        <View style={style.bottomContainer}>
-          <View>
-            <ShareLink />
-            <BluetoothActivationStatus />
-            <ProximityTracingActivationStatus />
-            <LocationActivationStatus />
-            {displayCovidData && <COVIDDataDashboard />}
+          <View style={style.statusBottomContainer}>
+            <Text style={style.statusActionText}>{actionText}</Text>
+            <SvgXml
+              xml={Icons.ChevronRight}
+              fill={Colors.black}
+              width={Iconography.tiny}
+              height={Iconography.tiny}
+            />
           </View>
-          <View style={style.ctaContainer}>
+        </TouchableOpacity>
+        <ShareLink />
+        {displayCovidData && <COVIDDataDashboard />}
+        {displayCallbackForm && (
+          <View style={style.floatingContainer}>
+            <Text style={style.sectionHeaderText}>
+              {t("home.did_you_test_positive")}
+            </Text>
+            <Text style={style.sectionBodyText}>
+              {t("home.to_submit_your_test")}
+            </Text>
             <Button
-              onPress={handleOnPressReportTestResult}
-              label={t("home.bluetooth.report_positive_result")}
+              onPress={handleOnPressTalkToContactTracer}
+              label={t("home.talk_to_a_contact")}
               customButtonStyle={style.button}
               customButtonInnerStyle={style.buttonInner}
               hasRightArrow
             />
-            {displaySelfAssessment && (
-              <Button
-                onPress={handleOnPressTakeSelfAssessment}
-                label={t("home.bluetooth.take_self_assessment")}
-                customButtonStyle={{
-                  ...style.button,
-                  ...style.selfAssessmentButton,
-                }}
-                customButtonInnerStyle={style.buttonInner}
-                outlined
-              />
-            )}
           </View>
+        )}
+        <View style={style.floatingContainer}>
```

Comment:
What do you think about `<ReportAPositiveTestClip />`?

Reply:
👍